src/stdlib_specialfunctions_gamma.fypp

```diff
@@ -1,7 +1,6 @@
-#:set WITH_QP = False
-#:set WITH_XDP = False
 #:include "common.fypp"
-#:set CI_KINDS_TYPES = INT_KINDS_TYPES + CMPLX_KINDS_TYPES
+#:set RC_KINDS_TYPES = REAL_KINDS_TYPES[0:2]
```

Comment:
For robustness, is it not possible to use something like:
```suggestion
#:set RC_KINDS_TYPES = REAL_KINDS_TYPES.remove("qp").remove("xdp")
```
If the order in `REAL_KINDS_TYPES` chnages (for whatever reasons), the current approach might be brokem

Reply:
I'll test this out, but not sure if it will work as is since `REAL_KINDS_TYPES` is the zipped varaible containing the 3 values INIT, KIND and TYPES. But it is true that if for whatever reason the order is changed that can be problematic.

Reply:
As I feared, this strategy does not work (it produces and error on fypp)